app/src/main/java/org/fossasia/susi/ai/skills/skilllisting/adapters/recycleradapters/SkillListAdapter.kt

```diff
@@ -21,18 +22,19 @@ class SkillListAdapter(val context: Context, private val skillDetails: List<Skil
         if (skillDetails != null) {
             val skillData = skillDetails.toTypedArray()[position]
 
-            if (skillData.skillName.isEmpty()) {
+            if (TextUtils.isEmpty(skillData.skillName)) {
                 holder.skillPreviewTitle.text = context.getString(R.string.no_skill_name)
             } else {
                 holder.skillPreviewTitle.text = skillData.skillName
             }
 
-            if (skillData.examples.isEmpty())
+            if (skillData.examples == null || skillData.examples.isEmpty()) {
```

Comment:
Is this one special?

Reply:
This one is List<String> , TextUtils was not working on it.
I found this approach in SkillsListAdapter